src/stash/src/upgrade/v33_to_v34.rs

```diff
@@ -156,13 +162,18 @@ async fn migrate_audit_log(tx: &mut Transaction<'_>, now: u64) -> Result<(), Sta
     // the new ID.
     let mut updates = Vec::with_capacity(live_replicas.len() * 2);
     for (_, details) in live_replicas {
-        let old_id = details.replica_id.unwrap().inner;
+        let old_id = details.replica_id.clone().unwrap().inner;
```

Comment:
nit: can you change this to an `.expect("filtered to only details with replica_ids");` or some similar error message? IIUC we are filtering only to details/audit log events that include replica_ids which makes this unwrap safe

Reply:
Yes! Done.